integrated_channels/integrated_channel/models.py

```diff
@@ -205,6 +206,76 @@ def clean(self):
                 }
             )
 
+    def get_learner_data_audit_model(self):
+        return LearnerDataTransmissionAudit.get_completion_class_by_channel_code(self.channel_code())
+
+    def get_last_learner(self, error, config_id):
```

Comment:
this is an instance method so, you don't need to pass in config_id, its already a variable available as `self.id`

nit: i'd maybe pick a clearer name - like `get_last_learner_synced_at` (maybe renamed `error` to `include_only_errors` or something like that)